site/public/js/manage-students.js

```diff
@@ -1,3 +1,4 @@
+/* global Cookies */
```

Comment:
It is probably better to declare `Cookies` as an eslint global in `.eslintrc.json` instead of labeling it as a global in every single file.